CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## UNRELEASED
+
+### Breaking changes
+* Removed arguments `privatekey1`, `privatekey2`, `privatekey3`, `publickey1`, `marketplacekey`
```

Comment:
I think you can not just remove them without providing another way to set keys for automatic testing. If I see it correctly tests are currently failing because of missing private key?